css/evaluations.css

```diff
@@ -0,0 +1,110 @@
+body{
+	background-color: white;
+}
+
+.logoClass{
+	background-image: url('https://www.softvision.ro/images/common/logo-v2.png');
+	width: 200px;
+	height: 37.40px;
+	background-size: contain;
+	margin-left: 50px;
+	margin-top: 20px;
+	float: left;
+	margin-right: 150px;
+}
+
+header{
+	width: 100%;
+	height: 80px;
+	box-shadow: 0px 3px 10px grey;
```

Comment:
Yup, just wanted to point that out. If you're more comfortable leaving the measurement unit in, no probz, just highlighting the fact that for `0`, measurement units are irrelevant. :)